opencensus/trace/utils.py

```diff
@@ -93,3 +95,40 @@ def disable_tracing_hostname(url, blacklist_hostnames=None):
             blacklist_hostnames = []
 
     return url in blacklist_hostnames
+
+
+def status_from_http_code(http_code):
+    """Returns equivalent status from http status code
+    based on OpenCensus specs.
+
+    :type http_code: int
+    :param http_code: HTTP request status code.
+
+    :rtype: int
+    :returns: A instance of :class: `~opencensus.trace.status.Status`.
+    """
+    grpc_code = None
+    if http_code <= 199:
+        grpc_code = code_pb2.UNKNOWN
+    elif http_code >= 200 and http_code <= 399:
+        grpc_code = code_pb2.OK
+    elif http_code == 400:
```

Comment:
Nit, potential way to simplify the flow.

```python
# validate that http_code is int

if http_code <= 199:
    return Status(code_pb2.UNKNOWN)

if http_code <= 399:
    return Status(code.pb2.OK)

grpc_code = {
    400: code_pb2.INVALID_ARGUMENT,
    401: code_pb2.UNAUTHENTICATED,
    # other code, sorted alphabetically
}.get(http_code, code_pb2.UNKNOWN)

return Status(grpc_code)

Reply:
Totally my fault, yours is way better, will change it.

Reply:
Done!